zsl/application/modules/context_module.py

```diff
@@ -0,0 +1,52 @@
+"""
+:mod:`zsl.application.modules.context_module`
+---------------------------------------------
+
+Application context describes the environment in which the application is run\
+and is responsible for specific initializations on it.
+"""
+from __future__ import unicode_literals
+
+from zsl.application.initialization_context import InitializationContext
+from injector import singleton, Module
+
+from zsl.application.initializers.library_initializer import LibraryInitializer
+from zsl.application.initializers.service_initializer import ServiceInitializer
+from zsl.application.initializers.unittest_initializer import UnitTestInitializer
+from zsl.application.initializers.web_initializer import WebInitializer
+
+#: Initializers used in all applications
+default_initializers = [LibraryInitializer, ServiceInitializer]
```

Comment:
I wanted them to be public, e. g. to be able to create some custom initialize list outside this package with these as base.